core-management/core-management-subsystem/src/test/java/org/wildfly/extension/core/management/CoreManagementSubsystem_1_0_TestCase.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.wildfly.extension.core.management;
+
+import org.jboss.as.subsystem.test.AbstractSubsystemSchemaTest;
+import org.jboss.as.subsystem.test.AdditionalInitialization;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.EnumSet;
+
+/**
+ * @author <a href="http://jmesnil.net/">Jeff Mesnil</a> (c) 2016 Red Hat Inc.
+ */
+@RunWith(Parameterized.class)
+public class CoreManagementSubsystem_1_0_TestCase extends AbstractSubsystemSchemaTest<CoreManagementSubsystemSchema_1_0> {
```

Comment:
Consider renaming this to w/out the version number, since this test case will support future versions, e.g. `CoreManagementSubsystemTestCase`